anaconda_project/project_commands.py

```diff
@@ -408,6 +408,8 @@ def description(self):
             description = self._attributes.get('windows', None)
         if description is None:
             description = self._attributes.get('conda_app_entry', None)
+        if description is None:
+            description = getattr(self, 'command', None)
```

Comment:
Mmm... in the plugins case `self.command` is really just a class attr. Is it a name issue (glad to change it in that case) or there was something else?

Overall I'd really like to merge this PR this morning.